simcon_project/simcon_project/urls.py

```diff
@@ -28,25 +28,31 @@
     path('accounts/', include('django.contrib.auth.urls')),
     path('redirect-from-login/', redirect_from_login, name="redirect-from-login"),
 
-    # Todo: remove
-    path('user-view/', researcher_user_view, name="researcher-user-view"),
-    path('studentuser-view/', student_user_view, name="student-user-view"),
-
     # Stuff students can see
     path('student/', student_view, name="student-view"),
-    path('student/settings/', student_settings_view, name="student-settings-view"),
-    path('student/register/<uidb64>/', student_registration, name="student-registration"),
-    path('student/conversation/', include('conversation_templates.urls.conv_urls'), name='conversation'),
+    path('student/settings/', student_settings_view,
+         name="student-settings-view"),
+    path('student/register/<uidb64>/',
+         student_registration, name="student-registration"),
+    path('student/conversation/',
+         include('conversation_templates.urls.conv_urls'), name='conversation'),
```

Comment:
I know its a little nit picky but would you be able to move these back to the single line like it was before?

Reply:
and the rest of them as well.

Reply:
i honestly cant, somehow prettier wont uninstall, I messed around with my settings and i cant get it to overwrite

Reply:
i fixed it, i ended up keybinding and shit..